processed 2 tasks

task 1 'publish'. lines 5-26:
Error: Transaction Effects Status: MiscellaneousError
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: ModuleVerificationFailure, source: Some("_::M1, 'init' function cannot have return values") } }
